Sources/ParseSwift/Objects/ParseUser.swift

```diff
@@ -699,8 +702,11 @@ extension ParseUser {
                     path: .users,
                     body: body) { (data) -> Self in
 
-            let sessionToken = try ParseCoding.jsonDecoder()
-                .decode(LoginSignupResponse.self, from: data).sessionToken
+            guard let sessionToken = try ParseCoding.jsonDecoder()
```

Comment:
Same here. When you add the necessary test cases, it will show this

Reply:
In fact I kept the original behavior, it has to fail if the response is malformed